lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -10,12 +10,18 @@ defmodule Membrane.MP4.Muxer.CMAF do
   """
   use Membrane.Filter
 
+  require Membrane.Logger
+
   alias __MODULE__.{Header, Segment}
   alias Membrane.{Buffer, Time}
-  alias Membrane.MP4.Helper
   alias Membrane.MP4.Payload.{AAC, AVC1}
+  alias Membrane.MP4.{Track, Helper}
+
+  def_input_pad :input,
+    availability: :on_request,
+    demand_unit: :buffers,
+    caps: Membrane.MP4.Payload
 
-  def_input_pad :input, demand_unit: :buffers, caps: Membrane.MP4.Payload
   def_output_pad :output, caps: Membrane.CMAF.Track
 
   def_options segment_duration: [
```

Comment:
I don't think it makes any difference here. From Membrane Core documentation:
> - type: atom, used for parsing
> - spec: typespec for value in struct. If ommitted, for types: [:atom, :boolean, :caps, :keyword, :string, :struct, :time] the default typespec is provided, for others typespec is set to any/0

Reply:
OK, I just noticed that in newer code we prefer using `spec` instead of `type`. Also, I think that for people not familiar with Membrane, `Membrane.Time.t()` would be more clear type indicator than bare `:time`.